### ISSUE TITLE
Support Dark mode on iOS/MacOS

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7783]
We should be supporting dark mode in the client for both iOS and MacOS for accessibility and consistency purposes. I came across a review on the Apple AppStore requesting the same feature a few months ago.

There is also a Foxfooding ticket requesting the same feature on Android: [https://mozilla-hub.atlassian.net/browse/VPN-3299](https://mozilla-hub.atlassian.net/browse/VPN-3299|smart-link)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-5398)
┆Reporter: Gela Malek Pour
